crafts.lua

```diff
@@ -110,25 +110,103 @@ core.register_craft({
 	recipe = { "default:dry_grass_1", "default:dirt" },
 })
 
+-- the density of ethereal mod charcoal is ~1/10 of coal, otherwise it's pure carbon
+-- the volume of charcoal mod charcoal is 1/8 of ethereal mod charcoal, since it uses planks for 2x the output count
+technic.register_compressor_recipe({ input = {"charcoal:charcoal_lump 80"}, output = "default:coal_lump 1" })
+
+technic.register_compressor_recipe({ input = {"factory_bridges:i00 30"}, output = "factory_bridges:b00 1" })
+technic.register_grinder_recipe({ input = {"factory_bridges:b00 10"}, output = "technic:wrought_iron_dust 3" })
```

Comment:
you have nice space after '{' and before '}' everywhere else, why stop here?

Reply:
also this needs to be wrapped in a clause checking that technic actually is being used.